### PR TITLE
Al 1323 test socify end end

### PR DESCRIPTION
**Pull request by rruster**
Fixing how data is pass to Socify.

### CODE REVIEW

File: disco_aws_automation/disco_deploy.py [hunk 1]
```diff
@@ -653,15 +653,15 @@ def test(self, dry_run=False, deployment_strategy=None, ticket_id=None):
...
                 raise
 
         socify_helper.send_event(
             status=status,
-            hostclass=(hostclass if ami else None),
-            previous_ami_id=(previous_ami_id if ami else None),
-            message=reason)
+            hostclass=(hostclass if ami else ""),
+            previous_ami_id=(previous_ami_id if ami else ""),
+            message=reason if reason else "")
```
Comment (jconstance-amplify) [inline disco_aws_automation/disco_deploy.py]: Could be `reason or ""`, which is a bit more concise. Same for the other ones.

File: disco_aws_automation/socify_helper.py [hunk 2]
```diff
@@ -126,6 +127,7 @@ def send_event(self, status, **kwargs):
             rsp_msg = response.json()['message']
             logger.info("received response status %s data: %s", status, rsp_msg)
         except requests.HTTPError:
+            logger.exception("Socify returns HTTPError")
```
Comment (jconstance-amplify) [inline disco_aws_automation/socify_helper.py]: Do we really need two error log messages here?

File: disco_aws_automation/socify_helper.py [hunk 3]
```diff
@@ -119,18 +120,20 @@ def send_event(self, status, **kwargs):
...
             status = response.status_code
             rsp_msg = response.json()['message']
-            logger.info("received response status %s data: %s", status, rsp_msg)
-        except requests.HTTPError:
-            rsp_msg = response.json()['errorMessage']
-            logger.error("Socify event failed with the following error: %s", rsp_msg)
-        except Exception:
-            logger.exception("Failed to send event to Socify")
-            rsp_msg = 'Failed sending the Socify event'
+            logger.info('received response status %s data: %s', status, rsp_msg)
```
Comment (jconstance-amplify) [inline disco_aws_automation/socify_helper.py]: Should this be a `debug` instead of `info`? Seems like it might be spammy.

Comment (rruster) [inline disco_aws_automation/socify_helper.py]: Will change all logger.info to logger.debug

**Pull request merged**